examples/simple/main.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2019 New Relic Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// This example script is the sample from the OpenTelemetry Go "Getting Started"
+// guide, with the text-based exporter replaced with the New Relic OpenTelemetry
+// Exporter.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/newrelic/newrelic-telemetry-sdk-go/telemetry"
+	"github.com/newrelic/opentelemetry-exporter-go/newrelic"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/baggage"
+	"go.opentelemetry.io/otel/label"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/metric/controller/push"
+	"go.opentelemetry.io/otel/sdk/metric/processor/basic"
+	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func main() {
+
+	// Create a New Relic OpenTelemetry Exporter
+	apiKey, ok := os.LookupEnv("NEW_RELIC_API_KEY")
+	if !ok {
+		fmt.Println("missing NEW_RELIC_API_KEY required for New Relic OpenTelemetry Exporter")
+	}
+
+	exporter, err := newrelic.NewExporter(
+		"Sample OpenTelemetry Service",
```

Comment:
The app name should be consistent with the folder name. We currently use `simple` for the folder and `sample` for the app name.

Reply:
Fixed, thanks.